tests/otel_integrations/test_openai.py

```diff
@@ -513,14 +648,186 @@ def test_sync_chat_empty_response_choices(instrumented_client: openai.Client, ex
                     'logfire.span_type': 'log',
                     'logfire.tags': ('LLM',),
                     'duration': 1.0,
-                    'response_data': '{"combined_chunk_content":"","chunk_count":0}',
+                    'response_data': '{"message":null,"usage":null}',
```

Comment:
why the change?

Reply:
This is now the natural response_data for when the response has an empty "choices" list because of the switch to piecing together the message. In this case there are no pieces.